tsconfig.json

```diff
@@ -10,6 +10,8 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "sourceMap": true,
+    // 忽略 this 的类型检查, Raise error on this expressions with an implied any type.
+    "noImplicitThis": false,
```

Comment:
Is this a must? I think to have this set to true is better.

Reply:
You are right,this rule set to true is better. I added this rule because there is an error in the src/utils/index.ts file when I use this in debounce return function. I will modify the code, and remove this rule.